webapp/src/components/EstateDetailPage/EstateDetailPage.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import EstateDetail from './EstateDetail'
+import Estate from 'components/Estate'
+import { isNewAsset } from 'shared/asset'
+import EditEstate from './EditEstate/EditEstate'
+import { parcelType } from 'components/types'
+
+export default class EstateDetailPage extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isEditing: props.isEditing,
+      isSelecting: props.isSelecting
+    }
+  }
+
+  static propTypes = {
+    x: PropTypes.number,
+    y: PropTypes.number,
+    assetId: PropTypes.string,
+    allParcels: PropTypes.objectOf(parcelType),
+    submitEstate: PropTypes.func.isRequired,
```

Comment:
this is missing the `on`, (should be `onSubmitEstate`), same thing with the line below, BUT i don't even see the point of having these props here, since they are not used here, just passed down to the `EditState` component, which is a container already so it could have these callbacks connected there, instead of passing down props down like this.